README.md

```diff
@@ -91,8 +91,8 @@ Download and install [Arduino IDE](https://www.arduino.cc/en/Main/Software). If
 
 You are almost ready to programme LightAPRS Tracker :)
 
-- First download the repository to your computer using the green "[clone or download](https://github.com/lightaprs/LightAPRS-1.0/archive/master.zip)" button.
-- There are more than one Arduino projects optimized for different use cases. For example if you are planning to use LightAPRS tracker for a pico balloon project, then use "[LightAPRS-pico-balloon](LightAPRS-pico-balloon)" folder, for a hab (high altitude balloon)/weather balloon project, then use "[LightAPRS-hab](LightAPRS-hab)" folder.
+- First download the repository to your computer using the green "[clone or download](https://github.com/VIP-LES/EosLightAPRS/archive/master.zip)" button.
+- We will use the items in the "LightAPRS-hab" folder, not the "LightAPRS-pico-balloon" or "LightAPRS-vehicle" folders. 
```

Comment:
Honestly could be worth just removing those directories entirely, unless you have plans to use them at some point.